### DISCUSSION TITLE
Disable toLowerCase for dbSafe fields

### DISCUSSION THREAD

**Discussion by grefrit** [directus/directus#9075] [Feature Requests]
Hello. I have database with camelCased fields and I can't change collection name normally. It seems to me that it would be better to allow large letters in the field.
https://github.com/directus/directus/blob/e3aa9fae2ac244e8f7742458cd4c4e3ced25484f/app/src/components/v-input/v-input.vue#L251

**Comment by rijkvanzanten** [comment]
If capitals are allowed in every supported database vendor, we can do this 👍🏻

**Comment by grefrit** [comment]
According to this [answer ](https://stackoverflow.com/a/44485142)all SQL databases case insensitive(must support camelCase). I tested PG, MySQL, MS SQL, SQLite and it's works. For [MariaDB](https://mariadb.com/kb/en/identifier-case-sensitivity/), [OracleDB](https://www.reddit.com/r/oracle/comments/1vh3ii/comment/ces82q6/?utm_source=share&utm_medium=web2x&context=3).

**Reply by grefrit** [reply]
@rijkvanzanten  Can I submit a PR to fix this issue?

**Reply by rijkvanzanten** [reply]
Yes, go ahead! Should be as simple as adding the uppercase characters in here:

https://github.com/directus/directus/blob/1b87e4555f44dd3b0c06d9a26bf4eef1c24f7805/app/src/components/v-input/v-input.vue#L210

**Reply by grefrit** [reply]
> Yes, go ahead! Should be as simple as adding the uppercase characters in here:
> 
> https://github.com/directus/directus/blob/1b87e4555f44dd3b0c06d9a26bf4eef1c24f7805/app/src/components/v-input/v-input.vue#L210

That's not necessary because of this line
https://github.com/directus/directus/blob/1b87e4555f44dd3b0c06d9a26bf4eef1c24f7805/app/src/components/v-input/v-input.vue#L181

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
